lib/virt_autotest/utils.pm

```diff
@@ -1486,4 +1497,11 @@ sub get_guest_settings {
     return \%settings_matrix;
 }
 
+# return 1 if UEFI boot should be enabled for VMs
+sub should_use_uefi {
+    # VIRT_LEGACY_BOOT=1 or not set means legacy mode (return 0)
+    # VIRT_LEGACY_BOOT=0 means UEFI mode (return 1)
+    return get_var("VIRT_LEGACY_BOOT", 1) ? 0 : 1;
```

Comment:
This is to decide/configure the VM boot mode, right? Compared to using a switch setting for all the guests, will it be more flexible to let each guest having their own boot mode? That is to configure the  "--boot firmware parameter" via https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/virt_autotest/common.pm.

Reply:
Fixed and controlled by common.pm